src/plugins/home/public/application/components/home_app.js

```diff
@@ -92,17 +108,25 @@ export function HomeApp({ directories, solutions }) {
           <Route exact path="/feature_directory">
             <FeatureDirectory addBasePath={addBasePath} directories={directories} />
           </Route>
-          <Route exact path="/">
-            <Home
-              addBasePath={addBasePath}
-              directories={directories}
-              solutions={solutions}
-              find={savedObjectsClient.find}
-              localStorage={localStorage}
-              urlBasePath={getBasePath()}
-              telemetry={telemetry}
-            />
-          </Route>
+          {homeConfig.disableNextHomePage ? (
```

Comment:
If disabled, shall we register the new homepage? From the consume logic, it is more like isNextHomePageDefaultIndexPage.

Reply:
Sure, just concern on if we should register the useless route.

Reply:
The route is not useless, as it needs to be viewable when "off" so that an admin or user can preview it.

Reply:
I see the confusion now. I've renamed it to something that better reflects its functionality.